src/lib.rs

```diff
@@ -7,7 +7,7 @@ pub static HOURS: i64 = 24;
 /// The Maximum number of bytes allowed in
 /// a request body
 ///
-pub static MAX_BODY: u64 = 20971520;
+pub static MAX_BODY: u64 = 20_971_520;
```

Comment:
I didn't know this was valid syntax, was this recommended by clippy?

Reply:
Yes, just to be more readable